python/apps/taiga/src/taiga/workflows/api/__init__.py

```diff
@@ -120,3 +108,53 @@ async def create_workflow_status(
         color=form.color,
         workflow=workflow,
     )
+
+
+################################################
+# update workflow status
+################################################
+
+
+@routes.stories.patch(
+    "/projects/{project_id}/workflows/{workflow_slug}/statuses/{status_slug}",
+    name="project.workflow.status.patch",
```

Comment:
I believe that the name should be "workflowstatus.update"

Reply:
You're right, it's a model instance and not a module. Thanks!
I'll change it also for other endpoints in this file.